Ssa why chrome why

trogdorsey opened the pull request:
If parent process is chrome, make sure the command line doesn't have `chrome-extension` on it.

Commit:
Chrome seems to call cmd.exe all the time for reasons. Removing it from the list.

Commit:
Added chrome back in, but filter out when the command line has chrome-extension in it

Commit:
Made the syntax more SPL2 friendly

Commit:
Forgot a lurking * in the LIKE statement

Commit:
Fixed more SPL2 errors

peter-cg reviewed (dismissed):
LGTM

Commit:
Added cmd_line to the body

peter-cg reviewed (dismissed):
Best detection update ever

Commit:
Added the data model

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

pyth0n1c committed:
Added a ba_detections.spec.json file for
validation of the BA detections.  Updated
validate_schema to use a different shcema
for BA detections vs non BA detections.

pyth0n1c committed:
Removed one line from json schema.
This should fail, removal and check for
failure checks that we fixed the underlying
issue.

pyth0n1c committed:
Just ba_detections to make parsing easier.

pyth0n1c committed:
some print statements during 
validation to help with debugging

pyth0n1c committed:
fixed checking of the ssa filename, I think

pyth0n1c committed:
more fixes to getting ssa detections

pyth0n1c committed:
Added Endpoint to ba_detections.spec.json

pyth0n1c committed:
Removed Endpoint, it should be Endpoint_Process
for relevant detections.

pyth0n1c committed:
Changed datamodel from Endpoint to Endpoint_Process
for a number of the ssa___ detection files.

patel-bhavin committed:
Branch was auto-updated.

pyth0n1c committed:
Added datamodels to most of the ssa___ detections.
Still needs some more attention - there are likely 
some missing datamodels and some that are incorrect.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

Commit:
Data model changes for ptt searches, bumped version of prohibited apps

Commit:
Merge branch 'ssa_why_chrome_why' of github.com:splunk/security-content into ssa_why_chrome_why

Commit:
Merge branch 'develop' into ssa_why_chrome_why

Commit:
Added some filters for the data model into the SPL

Commit:
Merge branch 'develop' into ssa_why_chrome_why

Commit:
improved ssa testing

P4T12ICK commented:
why do we need a new spec for ba detections?

Commit:
improved ssa testing

Commit:
small change

Commit:
small change

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

Commit:
Fixed data model

patel-bhavin committed:
Branch was auto-updated.

Commit:
Fixed data model

Commit:
Merge branch 'ssa_why_chrome_why' of github.com:splunk/security-content into ssa_why_chrome_why

Commit:
Merge branch 'develop' into ssa_why_chrome_why
